index.html

```diff
@@ -6145,83 +6145,85 @@ <h3>Algorithm</h3>
       in [[INFRA]].</p>
 
     <section id="internal-representation-in-implementations" class="informative">
-      <h4>Internal Representation in Implementations</h4>
-
-      <p>
-        <a data-cite="WebIDL#idl-USVString">USVString</a>,
-        <a>JsonLdRecord</a>,
-        <a data-cite="WebIDL#idl-sequence">sequence</a>,
-        and other types defined in [[WebIDL]] and [[INFRA]], and used above as input and output types for
-        <a>JsonLdProcessor</a> functions, comprise the data structure known
-        as the <a>internal representation</a>.
-        JSON-LD algorithms are defined against those types.
-      </p>
-
-      <p>
-        Implementations map [[WebIDL]] and [[INFRA]] types to types specific to the
-        programming language in use. For instance:
-      </p>
+      <ins cite="#change_6">
```

Comment:
Using `ins` here is not necessary, as this is an Informative section, so no normative text is changed. But, it can be referenced from the Candidate Change note.